modules/apps/transfer/types/denom_test.go

```diff
@@ -278,73 +260,52 @@ func (suite *TypesTestSuite) TestDenomChainSource() {
 	}{
 		{
 			"sender chain is source: empty trace",
-			types.Denom{
-				Base:  "uatom",
-				Trace: []types.Trace{},
-			},
+			types.NewDenom("uatom"),
```

Comment:
This is a nil trace, not an empty trace (which is tested in the next test case)

```suggestion
			types.NewDenom("uatom", []types.Trace{}),
```